singolo/index.html

```diff
@@ -0,0 +1,221 @@
+              </p>
+            </div>
+          </div>
+          <div class="list-items">
+            <div class="list-item-icon phone"></div>
+            <div class="list-item-text">
+              <h1>Responsive Layout</h1>
+              <p>
+                Sed porttitor placerat rhoncus. In at nunc tellus. Maecenas
+                blandit nunc ligula. Praesent elit leo.
+              </p>
+            </div>
+          </div>
+          <div class="list-items">
+            <div class="list-item-icon message"></div>
+            <div class="list-item-text">
+              <h1>24 / 7 Support</h1>
+              <p>
+                Vivamus vel quam lacinia, tincidunt dui non, vehicula nisi.
+                Nulla a sem erat. Pellentesque egestas venenatis lorem .
+              </p>
+            </div>
+          </div>
+          <div class="list-items">
+            <div class="list-item-icon star"></div>
+            <div class="list-item-text">
+              <h1>Feel Like A Star</h1>
+              <p>
+                Quisque hendrerit purus dapibus, ornare nibh vitae, viverra
+                nibh. Fusce vitae aliquam tellus.
+              </p>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="line-services"></div>
+    </section>
+    <section class="portfolio" id="portfolio">
+      <div class="portfolio-content">
+        <div class="portfolio-content__intro">
+          <h1 class="header-intro">Portfolio</h1>
+          <p class="text-intro">
+            Duis mollis, est non commodo luctus, nisi erat porttitor ligula,
+            eget lacinia odio sem nec elit.
+          </p>
+        </div>
+        <div class="portfolio-content__filter-list">
+          <div class="button-filter all"><p>All</p></div>
+          <div class="button-filter web-design"><p>Web Design</p></div>
+          <div class="button-filter graphic-design"><p>Graphic Design</p></div>
+          <div class="button-filter artwork"><p>Artwork</p></div>
+        </div>
+        <div class="porfolio-content__grid-images">
+          <div class="image-1 images"></div>
+          <div class="image-2 images"></div>
+          <div class="image-3 images"></div>
+          <div class="image-4 images"></div>
+          <div class="image-5 images"></div>
+          <div class="image-6 images"></div>
+          <div class="image-7 images"></div>
+          <div class="image-8 images"></div>
+          <div class="image-9 images"></div>
+          <div class="image-10 images"></div>
+          <div class="image-11 images"></div>
+          <div class="image-12 images"></div>
+        </div>
+      </div>
+      <div class="line-portfolio"></div>
+    </section>
+    <footer class="footer" id="footer">
+      <div class="footer-content">
+        <div class="footer-content__description"><p>© Copyright 2013</p></div>
+        <div class="footer-content__social-buttons">
+          <div class="button facebook"></div>
```

Comment:
I think it should be links that redirect to the social networks

Reply:
fixed